dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -20,7 +20,9 @@ class Api::V1::SectionsController < Api::V1::JsonApiController
   # Get the set of sections owned by the current user
   def index
```

Comment:
I (think) I found at least one other use of this route, so without knowing much about NetSim I'm inclined to keep the URL param:

https://github.com/code-dot-org/code-dot-org/blob/5af2e7f7286cde5c349d68ff30c1fe80f2b04103/apps/src/netsim/NetSimLobby.js#L165

Reply:
Update: we don't think student-level data is used by NetSim, so removing there as well.

For future references, `/dashboardapi/sections` and `/api/v1/sections` route to the same controller action because they use this shared "concern":

https://github.com/code-dot-org/code-dot-org/blob/5af2e7f7286cde5c349d68ff30c1fe80f2b04103/dashboard/config/routes.rb#L85

Reply:
TIL: `/dashboardapi/sections` and `/api/v1/sections` call into the same controllers.

Ben and I looked at this and think this instance also doesn't need the students list.